modules/napparameter/src/parameternumeric.h

```diff
@@ -109,5 +109,7 @@ namespace nap
 			RTTI_PROPERTY("Value",		&Type::mValue,			nap::rtti::EPropertyMetaData::Default)				\
 			RTTI_PROPERTY("Minimum",	&Type::mMinimum,		nap::rtti::EPropertyMetaData::Default)				\
 			RTTI_PROPERTY("Maximum",	&Type::mMaximum,		nap::rtti::EPropertyMetaData::Default)				\
+			RTTI_FUNCTION("setValue",	static_cast<void (Type::*)(decltype(Type::mValue))>(&Type::setValue))	\
```

Comment:
We should really start thinking about moving this to specialized interfaces for Python, we can't keep scattering around function definitions because you need them for this particular case. We either support it for a certain class or don't.

Reply:
I agree. There is the design document that I wrote about this:
https://docs.google.com/document/d/1QbV9nd_7S-vZH2Vq0V5KqjmP1FCYyfy5-ZzGA4Vd3Kw
For now: let's please support it for Parameter classes.